store/dataStore.ts

```diff
@@ -50,25 +51,43 @@ const initialState: IDataStoreState = {
   },
 };
 
+//create the store
```

Comment:
Q: I wonder if this comment is necessary when we're using the zustand create command here, which already implies we're creatings something, in this case the DataStore?

Reply:
For this comment, I was trying to implement something similar to what's currently used in apiFunctions.ts file. Instead of using `//`, I could use `/*** /` since JS Docs will be introduced shortly?

Reply:
Gotcha, that makes sense!